config/menus/options.cfg

```diff
@@ -1,6 +1,9 @@
+if (getalias selectedaspectratio) [] [ selectedaspectratio = 1 ]
```

Comment:
Replace with simply `selectedaspectratio = 1`
The alias will be overridden in config.cfg since it is `setpersist`ed.

Reply:
alright, will do